src/test/regress/expected/distributed_functions.out

```diff
@@ -39,7 +39,7 @@ CREATE TYPE dup_result AS (f1 int, f2 text);
 CREATE FUNCTION dup(int) RETURNS dup_result
     AS $$ SELECT $1, CAST($1 AS text) || ' is text' $$
     LANGUAGE SQL;
-SELECT create_distributed_function('dup(int)');
+SELECT create_distributed_function('dup(int)', '$1');
```

Comment:
I think we no longer have tests without distribution arguments, maybe restore these.